api/app/wildfire_one.py

```diff
@@ -667,3 +673,22 @@ def parse_hourly_actual(station_code: int, hourly_reading: WeatherReading):
 def validate_metric(value, low, high):
     """ Validate metric with it's range of accepted values """
     return low <= value <= high
+
+
+def parse_observation_valid_comment(comment):
```

Comment:
I'm not sure it's wise to trust an unstructured comment message to define the structured fields of concern. I think it's best to verify against the specific valid boolean fields the API supplies.

Reply:
but the API no longer supplies specific valid boolean fields?

Reply:
It supplies `observationValidInd`, any other validation on our end for specifics is fine. Relying on their comment would be building on shaky grounds, it could change out from under us at any time as there's nothing defined as to what it has to be besides a string.